books/free-programming-books-langs.md

```diff
@@ -2074,6 +2075,12 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [The Official Raspberry Pi Project Book 1 (2015)](https://magpi.raspberrypi.com/books/projects-1) (PDF)
 
 
+### RCS
+
+* [RCS — A System for Version Control](https://www.cs.purdue.edu/homes/trinkle/RCShome/rcs.ps) - Walter F. Tichy (PS)
+* [RCS Handbook](https://www.cs.purdue.edu/homes/trinkle/RCShome/contrib/odonavan/RCS_Book.tar.Z) - Brian O’Donovan
```

Comment:
Should we accept compressed files?

Reply:
There are some link that redirect to a compressed files so I think we accept those.

Example:
https://github.com/EbookFoundation/free-programming-books/blob/3d3929bc781106e4f18cd90d8f0867c8163dd8a9/books/free-programming-books-langs.md#L353